lib/pymedphys/_streamlit/apps/dashboard.py

```diff
@@ -23,7 +23,10 @@
 
 
 def main():
+    st.write("# Mosaiq QCL Dashboard")
+
     centres = ["rccc", "nbcc", "sash"]
```

Comment:
this will be much more broadly applicable if the centres and servers are read from configuration (falling back to these defaults is fine to save your own facility from having to do the configuration).

Reply:
It is currently unusable at other centres since it is not leaning on configuration. Just not as high priority as other items.

Reply:
Okay to approve given the scope of this PR?